lib/pacio_test_kit/common_tests/validation_test.rb

```diff
@@ -0,0 +1,50 @@
+require_relative '../pacio_profiles'
+require_relative '../interactions_test'
+
+module PacioTestKit
+  class ValidationTest < Inferno::Test
+    include PacioTestKit::PacioProfiles
+    include PacioTestKit::InteractionsTest
+
+    title 'Resources returned in previous tests conform to the given profile'
+    id :pacio_resource_validation
+    description %(
+      This test verifies resources returned from the read and search tests conform to
+      the [Pacio #{short_title}](#{PACIO_PROFILES[config.options[:profile]]}).
+
+      It verifies the presence of mandatory elements and that elements with required
+      bindings contain appropriate values.
+
+      This test will be skipped if no resource read or search requests were made in
+      previous tests or if all requests were unsuccessful.
+    )
+
+    def resource_type
+      config.options[:resource_type]
+    end
+
+    def tag
+      config.options[:profile]
+    end
+
+    def profile_url
+      PACIO_PROFILES[tag]
+    end
+
+    run do
```

Comment:
So as the validation_test is now, we only tell the user that resources of a resource_type are successful/validated etc. We don't mention which request it is? Example: If we read several id's, we just let the user know that only some of them returned were successful, but the user won't know the specific id if there is an error?

Reply:
The requests executed are displayed on the browser. The user will know which request failed. And if the validation fails for a resource, the validator will specify the resource with the given issue.

![Screenshot 2024-07-14 at 3 48 35 PM](https://github.com/user-attachments/assets/9a14103b-4670-4aaf-b997-a3662a7001a5)

![Screenshot 2024-07-14 at 3 50 16 PM](https://github.com/user-attachments/assets/317aee65-3d45-489c-9692-4f53a5bc6586)